src/Secrets.js

```diff
@@ -17,7 +17,7 @@ const MAIN_CHANNEL_USERNAME = "";
 const ADMINS = []
 
 
-const BOT_TOKEN = "819778324:AAEsM_p4hZbizY3zwz75uqhQooKBaMXSjMw";
```

Comment:
+ Token should be hidden

Reply:
It already been revoked.

Reply:
Ya i just noticed. Someone should merge your PR i think